### PR TITLE
Update eleven-labs setup README.md

### PR DESCRIPTION
**Pull request by lostviolinist**

### CODE REVIEW

File: typescript/examples/eleven-labs/conversational-agent/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This example uses the ElevenLabs base project described [here](https://elevenlabs.io/docs/conversational-ai/guides/conversational-ai-guide-nextjs) and follows the steps to add client tool calling described [here](https://elevenlabs.io/docs/conversational-ai/customization/client-tools).
 
-https://github.com/user-attachments/assets/67c7f2d2-0b6a-4e0a-b24b-423e35a94c69
+https://github.com/user-attachments/assets/2a53b2af-eefe-46c7-a3d4-9441eefe0f81
```
Comment (alfonso-paella) [inline typescript/examples/eleven-labs/conversational-agent/README.md:6]: Id explain what we're building. This is describing more how we build it, which is harder to parse

Sometihng on the lines of

This example builds a conversational agent with ElevenLabs, that can query the balance in your wallet and recommend you which tokens to buy, with some special touch ✨ 

It is built on top of the ElevenLabs base project, and implements both functionalities using tool calling.

File: typescript/examples/eleven-labs/conversational-agent/README.md [hunk 2]
```diff
@@ -11,12 +11,15 @@ https://github.com/user-attachments/assets/67c7f2d2-0b6a-4e0a-b24b-423e35a94c69
 ```
 cp .example.env .env
 ```
+`NEXT_PUBLIC_ELEVEN_LABS_AGENT_ID` - create an agent [here](https://elevenlabs.io/app/conversational-ai), and copy agent ID
```
Comment (alfonso-paella) [inline typescript/examples/eleven-labs/conversational-agent/README.md:14]: i didnt go through this, just double checking that we dont need to enable any special configs or anything

File: typescript/examples/eleven-labs/conversational-agent/README.md [hunk 3]
```diff
@@ -11,12 +11,15 @@ https://github.com/user-attachments/assets/67c7f2d2-0b6a-4e0a-b24b-423e35a94c69
...
 
-2. Create an ElevenLabs agent and get the agent ID. You can follow this guide: https://elevenlabs.io/docs/conversational-ai/docs/agent-setup
+`NEXT_PUBLIC_SEPOLIA_RPC_URL` - create your first app on [Alchemy](https://dashboard.alchemy.com/apps), and go to the Networks tab to copy Ethereum Sepolia API key
 
-3. Set the `NEXT_PUBLIC_ELEVEN_LABS_AGENT_ID` environment variable with the agent ID.
+`NEXT_PUBLIC_COINGECKO_API_KEY` - create your demo account API key on the Coingecko developer [dashboard](https://www.coingecko.com/en/developers/dashboard)
 
-4. ElevenLabs requires you to register each tool manually through the ElevenLabs dashboard. To make it easier, we've added a `logTools` option to the `getOnChainTools` function. This will log the tools with their respective descriptions and parameters to the console.
+2. Run `pnpm install` on the root directory (/goat). Then, go to the /conversational-agent directory to run `pnpm install && build`. After a successful build, run the app with `pnpm dev`, connect your wallet, and start the conversation!
```
Comment (alfonso-paella) [inline typescript/examples/eleven-labs/conversational-agent/README.md:20]: I think it's better to just put this as commands they can copy paste

Also, the first oen is incorrect, they need to be in the typescript/ folder, and they need to do `pnpm i && pnpm build`

File: typescript/examples/eleven-labs/conversational-agent/README.md [hunk 4]
```diff
@@ -11,12 +11,15 @@ https://github.com/user-attachments/assets/67c7f2d2-0b6a-4e0a-b24b-423e35a94c69
...
+
+**Note**: ElevenLabs requires you to register each tool manually through the ElevenLabs dashboard. To make it easier, we've added a `logTools` option to the `getOnChainTools` function. This will log the tools with their respective descriptions and parameters to the console.
```
Comment (alfonso-paella) [inline typescript/examples/eleven-labs/conversational-agent/README.md:22]: This i didnt get last time. 

Is it 

a) just purely FYI - that normally youd need to register tools but we solve it w a creative workaround

or

b) you do need to take action (if so, im unclear what)

No matter which is it, id make it more clear!

Comment (alfonso-paella) [inline typescript/examples/eleven-labs/conversational-agent/README.md:22]: I would not describe this as a note. I would make this step 3: Install tools in ElevenLabs console.

Also, in step 2, i would remove "run the app with `pnpm dev`, connect your wallet, and start the conversation!" as it wont work, until they do step 3, right?

Instead:
* step 3 could say "run pnpm dev and copy this output... blah"
* we can add a step 4 which is to actually run the agent, which says "run the app with `pnpm dev`, connect your wallet, and start the conversation!"

File: typescript/examples/eleven-labs/conversational-agent/README.md (hunk 2 above)
Comment (lostviolinist) [reply typescript/examples/eleven-labs/conversational-agent/README.md:14]: nope just make an agent and copy the ID

File: typescript/examples/eleven-labs/conversational-agent/README.md (hunk 4 above)
Comment (lostviolinist) [reply typescript/examples/eleven-labs/conversational-agent/README.md:22]: but if they dont run the app, they cant see the tools logged

**Comment by 0xaguspunk** [comment]
Is this still needed @lostviolinist ?

**Pull request closed without merging**